_docs/_latest/use/cli/explain.md

```diff
@@ -47,7 +49,7 @@ forseti model use <MODEL_NAME>
 
 Following are some example commands you can run to query the data model.
 
-### Listing resources in the data model
+### List resources in the data model
```

Comment:
Same comment as above, let's see if this is linked to.

Reply:
As above, redirect or persistent anchor?

Reply:
Resolved.